src/um/webauthn.rs

```diff
@@ -0,0 +1,221 @@
+// Licensed under the Apache License, Version 2.0
+// <LICENSE-APACHE or http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your option.
+// All files in the project carrying such notice may not be copied, modified, or distributed
+// except according to those terms.
+use shared::guiddef::GUID;
+use shared::minwindef::{BOOL, DWORD, PBYTE};
+use shared::windef::HWND;
+use um::winnt::{HRESULT, LONG, LPCWSTR, PCWSTR, PVOID};
+pub const WEBAUTHN_API_VERSION_1: DWORD = 1;
+pub const WEBAUTHN_API_VERSION_2: DWORD = 2;
+pub const WEBAUTHN_API_CURRENT_VERSION: DWORD = WEBAUTHN_API_VERSION_2;
+pub const WEBAUTHN_HASH_ALGORITHM_SHA_256: &'static str = "SHA-256";
+pub const WEBAUTHN_HASH_ALGORITHM_SHA_384: &'static str = "SHA-384";
+pub const WEBAUTHN_HASH_ALGORITHM_SHA_512: &'static str = "SHA-512";
```

Comment:
Nit:  `WEBAUTHN_HASH_ALGORITHM_SHA_*` doesn't match header order

Reply:
Should be fixed in 11662566dd9a2305be58503e2927cce8dbf42636.